src/vmm/src/vstate/vm.rs

```diff
@@ -296,26 +324,39 @@ impl Vm {
         self.get_irqchip()
             .restore_device(mpidrs, &state.gic)
             .map_err(RestoreStateError::GicError)
+
+        // TODO uncomment when `kvm_ioctls` is updated.
+        // let kvm = Kvm::new().map_err(VmError::Kvm)?;
+        // Self::check_extension(&kvm, &state.additional_capabilities)
```

Comment:
When are you going to address this TODO? If you don't have a clear plan, could you please create a tracking issue for this?

Reply:
Fixed